### PR TITLE
Update copyright notice

### LINKED ISSUE #454: Update copyright notice
At least the primary LICENSE file should be updated from
> Copyright (c) 2014 HashiCorp, Inc.

to a generic attribution/copyright holder, for example:

> Copyright (c) 2014-2023 HashiCorp, Inc.
> Copyright (c) 2023- OpenTofu contributors

See also https://reuse.software/spec/#format-of-copyright-notices

To avoid any confusion it could also be useful to explicitly state that contributors are not assigning or transferring any copyrights to HashiCorp.

### PR DESCRIPTION
**Pull request by namgyalangmo**
Resolves #454 

## Target Release

1.7.0

### CODE REVIEW

File: copyright_headers.go [hunk 1]
```diff
@@ -10,4 +12,3 @@ import (
 	_ "github.com/hashicorp/copywrite"
 )
 
-//go:generate go run github.com/hashicorp/copywrite headers
```
Comment (cube2222) [inline copyright_headers.go:13]: Is this removal intentional?

**Comment by cube2222** [comment]
Hey, thanks for working on this!

Could you please explain what the workflow would now be to invoke the copyright tool to generate headers in case new files are added?

File: copyright_headers.go (hunk 1 above)
Comment (namgyalangmo) [reply copyright_headers.go:13]: Yes, I realised line no. 13 wasn't adding any value to the code so I removed it.

**Comment by namgyalangmo** [comment]
> Hey, thanks for working on this!
> 
> Could you please explain what the workflow would now be to invoke the copyright tool to generate headers in case new files are added?

Sure, The workflow remains exactly the same, one just needs to run "copywrite headers" in terminal to get the copyrights in all the files with missing copyrights.  
PFA pic for reference. 
Screenshot 2024-02-01 at 12 21 54 AM

File: copyright_headers.go (hunk 1 above)
Comment (namgyalangmo) [reply copyright_headers.go:13]: in retrospect, line no.13 if commented out it will help automate the process. I will just that the line back for convenience!

**Review by cube2222 — APPROVED** [review]
LGTM!

I agree with the discussion in the issue, that the approach is kind of a hack. However, it's working (verified locally that `go generate` also works for me) and straightforward. I don't think it's worth spending more time on this to come up with a more "proper" approach.

**Comment by namgyalangmo** [comment]
> LGTM!
> 
> I agree with the discussion in the issue, that the approach is kind of a hack. However, it's working (verified locally that `go generate` also works for me) and straightforward. I don't think it's worth spending more time on this to come up with a more "proper" approach.

Great :) Will pick up another issue then!

**Comment by cube2222** [comment]
Actually, part of the pipeline doesn't pass @namgyalangmo.

It seems like `internal/cloudplugin/cloudproto1/cloudproto1.proto` shouldn't be changed. Copywrite intentionally omits .pb.go files, otherwise protobuf generation and copyright generation would race against each-other and yield inconsistent results.

In other words, please undo the change to that file.

**Comment by namgyalangmo** [comment]
> Actually, part of the pipeline doesn't pass @namgyalangmo.
> 
> It seems like `internal/cloudplugin/cloudproto1/cloudproto1.proto` shouldn't be changed. Copywrite intentionally omits .pb.go files, otherwise protobuf generation and copyright generation would race against each-other and yield inconsistent results.
> 
> In other words, please undo the change to that file.

On it!

**Comment by cube2222** [comment]
it also seems like there's a conflict with the main branch on the go.mod and go.sum files, that you'll need to resolve

**Comment by namgyalangmo** [comment]
> it also seems like there's a conflict with the main branch on the go.mod and go.sum files, that you'll need to resolve

will do.

**Comment by Yantrio** [comment]
@namgyalangmo , you do not need to keep closing and re-opening pull requests, you can continue to push and force-push to the same branch and it will update the PR in place.

**Comment by namgyalangmo** [comment]
> @namgyalangmo , you do not need to keep closing and re-opening pull requests, you can continue to push and force-push to the same branch and it will update the PR in place.

  hey Yantrio, I did not manually close this PR. I was trying to resolve the merge conflict and it closed on its own??

**Comment by Yantrio** [comment]
@namgyalangmo it looks like this branch now has no commits on it. Did you remove some commits i a rebase?

**Comment by namgyalangmo** [comment]
> @namgyalangmo it looks like this branch now has no commits on it. Did you remove some commits i a rebase?

Hey @Yantrio , I carefully went through the changes I made, seems like all my changes were reverted because I ran "git pull origin " in the terminal and it synced with the master branch. Because this entire fork has become really messy I was wondering if I could just create a new branch form scratch, make the suggested changes and push the PR again by EOD tomorrow?

**Comment by namgyalangmo** [comment]
I figured out how the PR got merged! 
Kuba had approved the changes, and the only thing that was stopping the pr from getting merged was the merge conflict.  When I accidentally reverted all the changes the merge conflict got resolved and the PR automatically got merged and then closed!

**Comment by cam72cam** [comment]
I'm sorry, but that's not what happened / not how this works.

**Comment by namgyalangmo** [comment]
> I'm sorry, but that's not what happened / not how this works.

Whoops.
